README.md

```diff
@@ -166,6 +193,18 @@ Videos from past workshops may be useful if you are following along independentl
 These can be found on the [ICCS youtube channel](https://www.youtube.com/@instituteofcomputingforcli3982) 
 under the 2023 Summer School materials.
 
+### Worked Solutions
+Worked solutions for all of the exercises can be found in the [worked solutions](worked-solutions/) directory.  
+These are for recapping after the course in case you missed anything, and contain ideal solutions complete with
+docstrings, outfitted with mypy type hints, linted, and conforming to the black code style.
```

Comment:
Hm, maybe add a link for each of these (docstrings, mypy type hints, linted, black code style), as people might have no clue what they mean, and this sentence might just confuse them?

Reply:
Damnit, I considered it whilst writing but felt lazy. I suppose this is why we have code reviews ;)

Reply:
OK, added links, is that OK?